examples/src/components/GeoPushpinChartConfigurationPointsSizeExample.jsx

```diff
@@ -0,0 +1,103 @@
+// (C) 2020 GoodData Corporation
+import React, { Component } from "react";
+import { GeoPushpinChart, Model } from "@gooddata/react-components";
+
+import "@gooddata/react-components/styles/css/main.css";
+
+import { projectId } from "../utils/fixtures";
+import { MAPBOX_TOKEN, cityCoordinatesUri, populationUri } from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const POINT_SIZE_OPTION = ["default", "0.5x", "0.75x", "normal", "1.25x", "1.5x"];
+
+export class GeoPushpinChartConfigurationPointsSizeExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            minSize: "default",
+            maxSize: "default",
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsSizeExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsSizeExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsSizeExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log(
+            "GeoPushpinChartConfigurationPointsSizeExample onCenterPositionChanged",
+            ...params,
+        );
+    }
+
+    onPointSizeChange = event => {
+        const { id, value } = event.target;
+        this.setState({
+            [id]: value,
+        });
+    };
+
+    renderPointSizeDropDown = id => (
+        <span style={{ display: "inline-block", minWidth: "10em", verticalAlign: "middle" }}>
+            {id}
```

Comment:
This visible text should be `Min Size` or `Max Size`, not actual value of `minSize | maxSize` so it would be more firendly to user.
Please help me double check that `id` can have space.

Reply:
I changed to **Smallest** and **Largest**.

Reply:
I changed to **Smallest** and **Largest**.

Reply:
I have pushed a `small fix` commit. Please refer to it.